web/reset/mail/index.php

```diff
@@ -143,9 +143,18 @@ function to64 ($v, $n)
     // Compare hashes
     if (!empty($v_hash)) {
         $salt = explode('$', $v_hash);
+        if($salt[0] == "{MD5}"){
         $n_hash = md5crypt($v_password, $salt[2]);
         $n_hash = '{MD5}'.$n_hash;
-
+        }else{
+            $v_password = escapeshellarg($v_password);
+            exec("doveadm pw -s ARGON2ID -p $v_password -t '$v_hash'", $output, $return_var);
```

Comment:
@jaapmarcus please port your fix to handle older distros here too (as you did in the shell scripts)